bucket/powertoys.json

```diff
@@ -21,6 +25,46 @@
             "Remove-Item \"$dir\\.tmp\", \"$dir\\$fname\" -Force -Recurse"
         ]
     },
+    "post_install": [
+        "'install-context', 'uninstall-context' | ForEach-Object {",
+        "    if (Test-Path \"$bucketsdir\\$bucket\\scripts\\$app\\$_.ps1\") {",
+        "        $content = Get-Content \"$bucketsdir\\$bucket\\scripts\\$app\\$_.ps1\"",
+        "        $content = $content.Replace('{{scoop_dir}}', $original_dir)",
+        "        if ($global) {",
+        "            $content = $content.Replace('{{install_scope}}', 'perMachine')",
+        "            $content = $content.Replace('{{registry_scope}}', 'HKLM')",
+        "        }",
+        "        $content = $content.Replace('{{install_scope}}', 'perUser')",
+        "        $content = $content.Replace('{{registry_scope}}', 'HKCU')",
+        "        $content | Set-Content -Path \"$dir\\$_.ps1\"",
+        "    }",
+        "}",
+        "$regpath = 'HKCU:\\Software\\Classes\\powertoys'",
+        "if ($global) {",
+        "    $regpath = $regpath.Replace('HKCU', 'HKLM')",
+        "}",
+        "if (Test-Path -Path $regpath) {",
+        "    Remove-Item -Path $regpath -Force | Out-Null",
+        "    \"$dir\\install-context.ps1\"",
+        "}"
+    ],
+    "uninstaller": {
+        "script": [
+            "$regpath = 'HKCU:\\Software\\Classes\\powertoys'",
+            "if ($global) {",
+            "    $regpath = $regpath.Replace('HKCU', 'HKLM')",
+            "}",
+            "if (Test-Path -Path $regpath) {",
+            "    \"$dir\\uninstall-context.ps1\"",
+            "    if ($cmd -ne 'uninstall') {",
+            "        New-Item -Path $regpath -Force | Out-Null",
+            "    }",
+            "}",
+            "if ($cmd -eq 'uninstall') {",
+            "    Stop-Process -Name explorer",
```

Comment:
This is intended because there are DLLs registered and used by `explorer`.

There are some other manifests are using this method.
Ref.
https://github.com/ScoopInstaller/Extras/blob/master/bucket/hashcheck.json#L27
https://github.com/ScoopInstaller/Extras/blob/master/bucket/folder-marker.json#L29